forge-gui/res/cardsfolder/upcoming/assemble_the_players.txt

```diff
@@ -0,0 +1,6 @@
+Name:Assemble the Players
+ManaCost:1 W
+Types:Enchantment
+S:Mode$ Continuous | Affected$ Card.TopLibrary+YouCtrl | AffectedZone$ Library | MayLookAt$ You | Description$ You may look at the top card of your library any time.
+S:Mode$ Continuous | MayPlay$ True | MayPlayLimit$ 1 | Affected$ Card.Creature+powerLE2+TopLibrary+YouCtrl | AffectedZone$ Library | Description$ Once each turn, you may cast a creature spell with power 2 or less from the top of your library.
```

Comment:
cast is not play

Reply:
This is fixed by a `nonLand` clause, right?